charts/edge/templates/configmap.yaml

```diff
@@ -13,3 +13,13 @@ data:
       request:
         headers:
           authorization: "Token ${MEZMO_LOCAL_DEPLOY_AUTH_TOKEN}"
+        {{- $payload := include "edge.heartbeatCommon" . | fromYaml }}
+        {{- $has_label := and (ge (.Capabilities.KubeVersion.Major | int) 1) (ge (.Capabilities.KubeVersion.Minor | int) 28) }}
+        minor: {{ .Capabilities.KubeVersion.Minor }}
+        has: {{ $has_label }}
+        {{- if $has_label }}
+        {{- $_ := set $payload "replica" "${POD_INDEX_LABEL}"}}
+        {{- else }}
+        {{- $_ := set $payload "replica" "0"}}
+        {{- end }}
+        paylaod: {{ toJson $payload | quote }}
```

Comment:
payload

Reply:
Good catch. Thanks.
I was working in the background to test this against the actual new vector, but happy to have it caught now.

Reply:
Fixed that and removed some lingering debug stuff.
Also got updated vector running with a config generated from this for e2e verification.